Signal/Images.xcassets/table_ic_notification_sound.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "table_ic_notification_sound@1x.png",
```

Comment:
IMO this looks too related to the "Mute" icon. What do you think about using music note?

Reply:
A capital idea.  What think you of this icon?

![simulator screen shot - iphone x - 2018-02-26 at 15 06 33](https://user-images.githubusercontent.com/625803/36692646-ab5bcc12-1b06-11e8-8a56-6098b37e6bbf.png)